pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -450,6 +450,18 @@ func (r *rbac) updateClusterRoleBinding(ctx context.Context, rb *rbacv1.ClusterR
 		rb.SetOwnerReferences([]metav1.OwnerReference{r.ownerRef})
 	}
 
+	doUpdateOwnerRef := true
```

Comment:
there are few other places in files where we check `len(ownerRef) ==0` and then add owner ref
while updating a resource.
should we add similar for those cases also?

Reply:
@nikhil-thomas

Reply:
@vinamra28 could you add a similar change in https://github.com/tektoncd/operator/blob/f061c934d3caf4b24ffa6f1d85f2d3658860d76c/pkg/reconciler/openshift/tektonconfig/rbac.go#L323